example.ini

```diff
@@ -30,6 +34,11 @@ factory = reddit_service_activitygateway:make_wsgi_app
 metrics.namespace = activity_gateway
 metrics.endpoint =
 
+; request tracing
+tracing.enabled = true
```

Comment:
after IRL discussion we're going to let the tracing integration always be on; if the endpoint is undefined here it'll just log with the Baseplate NullRecorder